app/dashboard/views.py

```diff
@@ -680,7 +680,16 @@ def sync_web3(request):
                 print('* getting bounty')
                 bounty = get_bounty(bounty_id, network)
                 print('* processing bounty')
-                did_change, _, _ = web3_process_bounty(bounty)
+                did_change = False
+                max_tries_attempted = False
+                counter = 0
+                while not did_change and not max_tries_attempted:
+                    did_change, _, _ = web3_process_bounty(bounty)
+                    if not did_change:
+                        print("RETRYING")
+                        time.sleep(3)
```

Comment:
I'm good with this, but 9 seconds might become bothersome for people if and when this happens. All in all 👍 though.

Reply:
itll retry every 3 seconds and will drop out of the loop if `did_change` == true at any of the iterations.

if we want to really hone the performance we could make this run 9x and only sleep 1 second each